common/artifacts/000_wc_artifacts_amulets_generic.txt

```diff
@@ -0,0 +1,370 @@
+### Amulets ###
+
+# Legendary
+
+amulet_5_diplomacy = {
+	quality = 5
+	flags = { amulet }
+
+	diplomacy = 3
+
+	general_opinion = 4
+	vassal_opinion = 4
+	same_religion_opinion = 8
+
+	monthly_character_prestige = 1
+
+	picture = "GFX_inv_6_0raid_necklace_2a"
+
+	slot = neck
+}
+
+amulet_5_martial = {
+	quality = 5
+	flags = { amulet }
+
+	martial = 3
+
+	monthly_character_prestige = 1
+
+	command_modifier = {
+		speed = 0.1
+		siege = 0.05
+	}
+
+	global_levy_size = 0.2
+
+	picture = "GFX_inv_6_0raid_necklace_2b"
+
+	slot = neck
+}
+
+amulet_5_stewardship = {
+	quality = 5
+	flags = { amulet }
+
+	stewardship = 3
+
+	global_tax_modifier = 0.05
+	build_cost_modifier = -0.1
+	build_time_modifier = -0.075
+
+	monthly_character_prestige = 1
+
+	picture = "GFX_inv_6_0raid_necklace_2c"
+
+	slot = neck
+}
+
+amulet_5_intrigue = {
+	quality = 5
+	flags = { amulet }
+
+	intrigue = 3
+
+	plot_power_modifier = 0.2
+	murder_plot_power_modifier = 0.4
+	plot_discovery_chance = 0.2
+
+	monthly_character_prestige = 1
+
+	picture = "GFX_inv_6_0raid_necklace_2p"
+
+	slot = neck
+}
+
+amulet_5_learning = {
+	quality = 5
+	flags = { amulet }
+
+	learning = 3
+	health = 0.3
+	fertility = 0.2
+
+	monthly_character_prestige = 1
+	monthly_character_piety = 0.25
```

Comment:
Cost for prestige+piety should be equal to 10, according to zumbak's rule. Means, either reduce prestige here by .5, reduce prestige and piety or eliminate piety. At the same time, increase other values, because right now it's a perfect C/Q of 5.00

Reply:
However, balancing could be done in the next branch, I think,

Reply:
> Cost for prestige+piety should be equal to 10, according to zumbak's rule.

Yeah, sure

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/70134247-bd994500-1698-11ea-909c-9d28e9db2efa.png)
![image](https://user-images.githubusercontent.com/46576860/70134263-c722ad00-1698-11ea-8d89-1fb3d43f670b.png)
![image](https://user-images.githubusercontent.com/46576860/70134324-e7eb0280-1698-11ea-88fc-0d56828827f6.png)

</details>

Reply:
That's one of those to be balanced soon. It's still at 1.5 prestige equivalents...

Reply:
Changed